extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -475,18 +567,24 @@ public function getArticleMatchForTermAndNamespaces( $term, array $namespaces )
 				$page = $this->getPageFromPageId( $articleId );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$page" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)